tapir/core/apps.py

```diff
@@ -73,3 +73,10 @@ def register_sidebar_link_groups():
             url=reverse_lazy("coop:about"),
             ordering=7,
         )
+
+        misc_group.add_link(
+            display_name=_("Become a Member"),
+            material_icon="person_add",
+            url=reverse_lazy("coop:member_self_registration"),
+            ordering=8,
+        )
```

Comment:
Maybe we could show that link only if the person is not logged in?

The sidebar menu code is a bit rudimentary but it should be possible check for that, see `tapir.core.utils.SidebarLinkGroups.is_link_shown`